### ISSUE TITLE
About the data sets

### ISSUE THREAD

**Issue by zhongming12342** [PredictiveIntelligenceLab/cvit#7]
Hi, Thank you for your work! 

The question is about the dataset of shallow water wave equations. The dataset of shallow water wave equations downloaded from PDEarena has 7000 training sets and 1400 test sets (I used DPOT's code to extract the data individually, and it saved the validation set in PDEarena as the training set), but your article mentioned 5600 training sets and 1000 test sets. 
Did you only extract part of it?
And it seems that you packaged these data and saved them in a 'zarr' file?

**Comment by sifanexisted** [comment]
Hi,

Thank you for reaching out. The reason is simply that the PDEArena dataset was updated. Originally, it contained 5,600 trajectories and was stored in Zarr format.

Best,
the authors

**Comment by zhongming12342** [comment]
OK! Thank you for your reply!
